docs/advisors/checks/mongo-high-connections.md

```diff
@@ -0,0 +1,42 @@
+# MongoDB High Connections
+
+## Description
+This check returns warnings if there is an increase in the number of connections.
```

Comment:
```suggestion
This check warns if the number of connections is increasing.
```

Reply:
I think, the above statement should be -
"This check warns if the number of connections are increased."

Reply:
Hmm, the agreement is with "the number" here.